### DISCUSSION TITLE
How to access member clusters through karmada?

### DISCUSSION THREAD

**Discussion by GitHubxsy** [karmada-io/karmada#1077] [Ideas]
Users can easily access member clusters through karmada. And supports all APIs of member clusters .Such as get, create, update, etc.
Similar issue #451

**Comment by GitHubxsy** [comment]
We can achieve cluster/proxy capabilities through [AA（apiserver-aggregation）.](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation/) As shown below：
![image](https://user-images.githubusercontent.com/11901080/144958346-fafa2c9a-c122-4305-a754-39ae6bd23397.png)

![image](https://user-images.githubusercontent.com/11901080/145360031-d2930c3f-4a04-4545-bf80-9264f73efe92.png)

proxy api：**`/apis/cluster.karmada.io/v1alpha1/cluster/{clustername}/proxy/api/v1/nodes`**
The proxy api is divided into three parts：

1. Specify cluster：for example, cluster1
2. proxy： Fixed value
3. proxyurl


reference：

- [ apiserver-proxy](https://kubernetes.io/zh/docs/tasks/access-application-cluster/access-cluster/#%E5%8F%91%E7%8E%B0%E5%86%85%E5%BB%BA%E6%9C%8D%E5%8A%A1)

**Comment by XiShanYongYe-Chang** [comment]
Our feature is ready now, we can refer user guide here:

https://github.com/karmada-io/karmada/blob/master/docs/userguide/aggregated-api-endpoint.md

**Comment by chaunceyjiang** [comment]
proxy api：/apis/cluster.karmada.io/v1alpha1/clusters/{clustername}/proxy/api/v1/nodes 

https://github.com/karmada-io/karmada/blob/master/pkg/aggregatedapiserver/apiserver.go#L106-L108
